_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,270 @@
+  accept transactions with zero confirmations ("zeroconf") as final
+  payments.  Discussion continued this week on both the mailing list and
+  the #bitcoin-core-dev IRC room.  Some highlights of the discussion
+  include:
+
+    - *Free option problem:* Sergej Kotliar [warned][kotliar free
+      option] that he believes the greatest problem with transaction
+      replacement is that it creates a free American call option.  For
+      example, customer Alice requests to buy widgets from merchant Bob.
+      Bob gives Alice an invoice for 1 BTC at the current price of
+      $20,000 USD/BTC.  Alice sends Bob the 1 BTC in a transaction with
+      a low feerate.  The transaction remains unconfirmed when the
+      exchange rate changes to $25,000 USD/BTC, meaning Alice is now
+      paying $5,000 more.  At this point, she quite rationally chooses
+      to replace her transaction with one paying the BTC back to
+      herself, effectively canceling the transaction.  However, if the
+      exchange rate changed in Alice's favor (e.g. $15,000 USD/BTC), Bob
+      can't cancel Alice's payment and so he has no way in the normal
+      onchain Bitcoin transaction flow to exercise the same option,
+      creating an asymmetric exchange rate risk.  By comparison, when
+      transaction replacement isn't possible, Alice and Bob share the
+      same exchange rate risk.
+
+        Greg Sanders and Jeremy Rubin [replied][sanders cpfp] in
+        [separate][rubin cpfp] emails to note that merchant Bob could
+        incentivize miners to confirm customer Alice's original
+        transaction using [CPFP][topic cpfp], particularly if [package
+        relay][topic package relay] was enabled.
+
+        Antoine Riard [noted][riard free option] that the same risk
+        exists with LN, as Alice could wait to pay merchant Bob's
+        invoice up until shortly before it expired, giving her time to
+        wait for the exchange rate to change.  Although in that case, if
+        Bob noticed that the exchange rate had changed significantly, he
+        could instruct his node not to accept the payment, returning the
+        money to Alice.
+
+    - *Bitcoin Core not in charge of network:* Gloria Zhao [wrote][zhao
+      no control] in the IRC discussion, "I think whatever option we
+      take, it should be made abundantly clear to users that Core does
+      not control whether full RBF happens or not. We could revert
+      [25353][bitcoin core #25353] and it could still happen. [...]"
+
+    - *No removal means problem could happen:* in the IRC discussion,
+      Anthony Towns [echoed][towns uncoordinated] his points from last
+      week, "if we're not going to remove the mempoolfullrbf option from
+      24.0, we're going for an uncoordinated deployment."
+
+        Greg Sanders was [doubtful][sanders doubt], "the question is
+        will 5%+ set a variable. I suspect not."  Towns [replied][towns
```

Comment:
I'm not sure how far we want to edit direct quotes, but the “is will” throws me off a bit. Perhaps it could be smoothed out by adding a “[whether]” or comma, or this?

```suggestion
        Greg Sanders was [doubtful][sanders doubt], "the question is
        'will 5%+ set a variable?' I suspect not."  Towns [replied][towns
```